extensions/binding/org.eclipse.smarthome.binding.homematic/README.md

```diff
@@ -223,6 +225,15 @@ The disadvantage is of course, that all events for this channel are delayed.
 ```
 
 The Type is the device type, channel number and lowercase channel name separated with a underscore.
+If you are using Homegear, the ```HG-``` prefix is not needed for the Type.
```

Comment:
I think that one could make more explicit why this is explicitly mentioned for Homegear here (for readers who have not read all the above). Suggestion: "Note that, for Homegear devices, in contrast to the specification of the Thing above no ```HG-``` prefix is needed for the specification of the Type of the Channel.